service.py

```diff
@@ -18,7 +18,7 @@
 app.secret_key = 'A0Zr98j/3yX R~XHH!jmN]LWX/,?RT'
 app.config['SEND_FILE_MAX_AGE_DEFAULT'] = timedelta(seconds=1)
 db = SQLAlchemy(app)
-IP = '168.0.0.10'
+IP = '127.0.0.1'
```

Comment:
@Weichao-Chen 

What is the point of changing the IP address?  Add a comment for that.


Hui